### DISCUSSION TITLE
Hide current event

### DISCUSSION THREAD

**Discussion by loryanstrant** [totaldebug/atomic-calendar-revive#806] [General]
Is there a way to hide the current event from the list?
I've created a card that shows only the current event, and a separate card that shows the rest of the day.
I'd like the rest of the day card to not show the current event, so it's not shown twice.
![image](https://user-images.githubusercontent.com/51473494/193711583-07ba3acb-2e8f-48ad-9dd6-d258d5a7a233.png)

**Comment by marksie1988** [comment]
No this is not possible
